fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -1719,8 +1724,13 @@ private void handleShowData() {
                     Pair<Double, String> tableSizePair = DebugUtil.getByteUint(tableSize);
                     String readableSize = DebugUtil.DECIMAL_FORMAT_SCALE_3.format(tableSizePair.first) + " "
                             + tableSizePair.second;
+                    Instant instant = Instant.ofEpochMilli(olapTable.lastVersionUpdateEndTime.get());
```

Comment:
Maybe I didn't express myself clearly enough before, here we want to show the modification time of the data size, i.e, when the result of `olap.getDataSize()` was updated, not the modification time of the partition version ^_^

Reply:
You can reference 
https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/catalog/TabletStatMgr.java to see how the data size was updated.

Reply:
@sduzh Hello,  I am a new user of SR and I am not clear about the concept of `version`. Dosen't it  increase after data updating? Or we should record the time after transaction applied?

Reply:
@lingtaolf Sorry for the late reply. The version number will be increased after the new data is imported successfully, but the stats seen by the `show data` command will not be updated immediately after the data is imported. The results of the `show data` command are collected regularly in the background, so they are not synchronized with the update time of the version.